### PR TITLE
[FIX]: DiggerComponent use_efficiency property added

### PR DESCRIPTION
**Pull request by Refaltor77**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/
 vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+PocketMine-Maa.phar
```
Comment (dadodasyra) [inline .gitignore:4]: This has nothing to do here

File: src/item/component/DiggerComponent.php [hunk 2]
```diff
@@ -18,7 +18,8 @@ public function getName(): string {
 
 	public function getValue(): array {
 		return [
-			"destroy_speeds" => $this->destroySpeeds
+			"destroy_speeds" => $this->destroySpeeds,
+            "use_efficiency" => true
```
Comment (dadodasyra) [inline src/item/component/DiggerComponent.php:22]: you should use the same indentation has customies (tabulations not spaces)

File: src/item/component/IconComponent.php [hunk 3]
```diff
@@ -18,8 +18,8 @@ public function getName(): string {
 	public function getValue(): array {
 		return [
 			"textures" => [
-				"default" => $this->texture
-			]
+                'default' => $this->texture
```
Comment (dadodasyra) [inline src/item/component/IconComponent.php:21]: customies uses " and not ' everywhere in it's code, changing that makes no sense and is decreasing code quality (+ indentation is still tabulations)

**Review by dadodasyra — CHANGES_REQUESTED** [review]

**Comment by dadodasyra** [comment]
What is your code changing ? what's the purpose of adding this value ?

**Comment by TwistedAsylumMC** [comment]
Closing due to changing other stuff that's irrelevant

**Pull request closed without merging**